tipsy/SettingsViewController.swift

```diff
@@ -10,41 +10,35 @@ import UIKit
 
 class SettingsViewController: UIViewController {
 
-    @IBOutlet weak var defaultTipControl: UISegmentedControl!
+    @IBOutlet private weak var defaultTipControl: UISegmentedControl!
+    
+    private let defaults = UserDefaults.standard
+    
+    // MARK: - Lifecycle Methods
```

Comment:
I used `// MARK: - ` to organize my code.

![organized file](https://user-images.githubusercontent.com/5225621/28857768-887508bc-7700-11e7-87cb-ed57595054b6.png)

Notice the separate bolded sections in the dropdown.